ArticleTemplates/assets/js/modules/messenger.js

```diff
@@ -0,0 +1,198 @@
+/* global Promise */
```

Comment:
😭  one question I wanted to ask you is the policy around support strategy. This is your call, I can work around that by using callbacks instead if you feel it is safer.